swarm.go

```diff
@@ -19,6 +19,13 @@ type SwarmConfig struct {
 	// `Swarm.Transports.Relay` if specified.
 	DisableRelay bool `json:",omitempty"`
 
+	// EnableRelayHop makes this node act as a public relay, relaying
+	// traffic between other nodes.
+	//
+	// Deprecated: The circuit v1 protocol is deprecated.
+	// Use `Swarm.RelayService` to configure the circuit v2 relay.
+	EnableRelayHop bool
```

Comment:
Maybe we should make this a `Flag` with `omitempty`, so that this won't created in newly initialized config files?

Reply:
No, Flag is int8. Let's keep it as bool and just add omitempty:

```suggestion
	EnableRelayHop bool `json:",omitempty"`
```